plane/source/docs/automatic-landing.rst

```diff
@@ -75,69 +102,6 @@ You should also be aware that many model aircraft can glide for long
 distances, and it may be that your requested glide slope and airspeed
```

Comment:
I guess you didn't change this but I think "TECS tuning patch" should be "TECS tuning page" and it would probably be useful to link to that page.